rust_snuba/src/processors/uptime_monitor_checks.rs

```diff
@@ -146,4 +146,50 @@ mod tests {
         assert_eq!(monitor_row.retention_days, 30);
         assert_eq!(timestamp, 1702659277.0);
     }
+
+    #[test]
+    fn test_parse_monitor_checkin_null_duration() {
+        let data = r#"{
+            "organization_id": 1,
+            "project_id": 1,
+            "retention_days": 30,
+            "region": "global",
+            "environment": "prod",
+            "subscription_id": "123e4567-e89b-12d3-a456-426614174000",
+            "guid": "550e8400-e29b-41d4-a716-446655440000",
+            "scheduled_check_time_ms": 1702659277,
+            "actual_check_time_ms": 1702659277,
+            "duration_ms": null,
+            "status": "ok",
```

Comment:
I think duration is only null for misses, should we change this to a miss to better reflect when duration would be null?

https://github.com/getsentry/sentry-kafka-schemas/blob/fddec4d390658d8177d46da6de1176f2e4534f77/schemas/uptime-results.v1.schema.json#L98

Reply:
i don't think it really matters for the sake of the test, but sure.

Reply:
I think it's always good to try and make tests reflect reality as much as possible (within reason), since it can really help to avoid confusion.

Reply:
updated!